# Polaris Parameters
lookAhead: 10.0
wheelBase: 2.4
steeringRatio: 12.0
steerLim:
 lower: -7.5
 upper: 7.5
# PID GAINS
# large error throttle
PID1:
 P: 0.015
 I: 0.004
 D: 0.01
# large error brake
PID2:
 P: 0.0001
 I: 0.
 D: 0.
# small error brake
PID3:
 P: 0.00005
 I: 0.
 D: 0.
# Tolerance
Tol: 0.1


